pypicloud/storage/__init__.py

```diff
@@ -4,6 +4,7 @@
 from .base import IStorage
 from .files import FileStorage
 from .s3 import S3Storage, CloudFrontS3Storage
+from .gcs import GCSStorage
```

Comment:
If you notice the tests are failing because they can't find `gcs_oauth2_boto_plugin`. You'll need to add that to the `requirements_build.txt` file, but you should also change around this import. In order for pypicloud to run without that plugin (which it should be able to), we can't import `GCSStorage` here. I think the easiest way to fix this would be to omit this import, and below on line 19 change the import path to `pypicloud.storage.gcs.GCSStorage`.